helper-hardhat-config.ts

```diff
@@ -10,28 +10,37 @@ dotenv.config({});
 
 const INFURA_KEY = process.env.INFURA_KEY || '';
 const ALCHEMY_KEY = process.env.ALCHEMY_KEY || '';
+const QUICKNODE_URL = process.env.QUICKNODE_URL || '';
 const TENDERLY_FORK_ID = process.env.TENDERLY_FORK_ID || '';
 
 const GWEI = 1000 * 1000 * 1000;
 
 export const NETWORKS_RPC_URL: iParamsPerNetwork<string> = {
   [eEthereumNetwork.kovan]: ALCHEMY_KEY
     ? `https://eth-kovan.alchemyapi.io/v2/${ALCHEMY_KEY}`
-    : `https://kovan.infura.io/v3/${INFURA_KEY}`,
+    : `https://kovan.infura.io/v3/${INFURA_KEY}`
+    ? `https://kovan.infura.io/v3/${INFURA_KEY}`
+    : `${QUICKNODE_URL}`,
```

Comment:
Wont ``https://kovan.infura.io/v3/${INFURA_KEY}`` always return truthy?

```suggestion
    : (INFURA_KEY
       ? `https://kovan.infura.io/v3/${INFURA_KEY}`
       : `${QUICKNODE_URL}`),
```

Reply:
It would yes. pushing up a change to fix